app/views/questionnaire.py

```diff
@@ -273,6 +274,7 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
             routing_path,
         )
 
+        message = json.dumps(message)
```

Comment:
I think that this should be done within `convert_answers` and `convert_answers` should now return a string.
Its not best practice to re-set a variable.